addons/l10n_tr/data/account_tax_template_data.xml

```diff
@@ -19,7 +19,7 @@
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'tax',
-                'account_id': ref('tr391'),
+                'account_id': ref('acc391'),
```

Comment:
Don't change the xml id prefix plz because sometimes, it's useful to retrieve the existing mapping between `account.account.template`& `account.account` during some migrations and if you change this prefix, this mapping is gone forever

Reply:
The reason is there are lots of other numeric codes and `tr`  was very confusing, but i'll do it .